src/redmine.coffee

```diff
@@ -259,27 +253,38 @@ module.exports = (robot) ->
       _.push "\n#{issue.description}"
 
       # journals
-      _.push "\n" + Array(10).join('-') + '8<' + Array(50).join('-') + "\n"
-      for journal in issue.journals
-        do (journal) ->
-          if journal.notes? and journal.notes != ""
-            date = formatDate journal.created_on, 'mm/dd/yyyy (hh:ii ap)'
-            _.push "#{journal.user.name} on #{date}:"
-            _.push "    #{journal.notes}\n"
+      if issue.journals?
+        _.push "\n" + Array(10).join('-') + '8<' + Array(50).join('-') + "\n"
+        for journal in issue.journals
+          do (journal) ->
+            if journal.notes? and journal.notes != ""
+              date = formatDate journal.created_on, 'mm/dd/yyyy (hh:ii ap)'
+              _.push "#{journal.user.name} on #{date}:"
+              _.push "    #{journal.notes}\n"
 
       msg.reply _.join "\n"
 
   # Robot redmine search <query>
   robot.respond /redmine search (.*)/i, (msg) ->
     params =
-      q: encodeURI msg.match[1]
+      q: msg.match[1]
       limit: process.env.HUBOT_REDMINE_SEARCH_LIMIT || 10
-    redmine.search params, (err, data) ->
-      if err?
-        msg.reply "Couldn't get search results!"
-        robot.logger.debug err
-      else
-        if data.total_count?
+
+    # Search endpoint Not available in node-redmine@0.2.1
```

Comment:
Ideally this would be through the API harness as well, but it is not currently available (seems to have lacked documentation at the time).